autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -190,7 +191,7 @@ def add_arguments(self, parser):
         parser.add_argument('--year', '-y', type=int,
                             help="A year to scrape all courses for, such as 2019")
 
-    def handle(self, *args, **options): # pylint: disable=too-many-locals, too-many-statements
+    def handle(self, *args, **options): # pylint: disable=too-many-locals, too-many-statements, too-many-branches
```

Comment:
I feel like we should split up the logic here into a few different functions instead of ignoring the linter since this function is getting really long, `get_course_data()` for the asyncio loop + making the model arrays and `save_models()` for all of the logic for saving seems like a good way to do it

Reply:
Yeah I figured as much, I think I was just being lazy at the time